### DISCUSSION TITLE
Probe Dragging?

### DISCUSSION THREAD

**Discussion by CarbonLancer** [jneilliii/OctoPrint-BedLevelVisualizer#380] [Q&A]
Hi, everything seems to be working as normal, BUT the probe is dragging across the bed to the next point, raising up, dropping, then dragging again?

The only code I am using:
G28
G29 T 

and Use Relative Z Offsets is selected. 

I am using a Chiron.

**Comment by jneilliii** [comment]
The plugin doesn't do anything relative to the probing process, so maybe try with the commands listed on the examples [here](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/blob/master/wiki/gcode-examples.md#example-gcode) based on your firmware and leveling type? Your firmware might not like the `T` parameter for the G29 and/or you should put the @BEDLEVELVISUALIZER flag just prior to the G29 command at minimum.
